torchci/pages/api/drci/drci.ts

```diff
@@ -867,6 +868,11 @@ export async function getWorkflowJobsStatuses(
         continue;
       }
 
+      if (isExcludedFromFlakiness(job)) {
```

Comment:
IMO, we should keep `isInfraFlakyJob` before this, as that categories is a legit flaky one if it happens.  Excluding flakiness check there would imply that infra is 100% reliable, which is not the case.  The rest of the order looks fine

Reply:
I think the idea in that case is that we want to force the developer to either rerun the job or explicitly bypass it

Reply:
Well, they indeed should rerun infra flaky job but my observation is that most people will just yolo bypass it and complain that infra is flaky :)